content/api-streamer/overview.textile

```diff
@@ -0,0 +1,39 @@
+---
+title: Overview
+section: api-streamer
+index: 65
+---
+
+h2(#what-is-api-streamer). What is API Streamer?
+
+"API Streamer":https://ably.io/api-streamer makes it easy and cost-effective to expose data feeds and create realtime APIs. It combines Ably's Data Stream Network with realtime API management tools to provide the first unified platform for managing the full lifecycle of realtime APIs - from deployment to distribution.
+
+Typically today, data producers provide rate-limited REST APIs for developers to consume data from. But increasingly those APIs are failing to meet developer performance expectations, as polling a REST API offers substandard latencies. So data producers are creating realtime APIs to meet developer demand for realtime data.
+
+However, creating realtime APIs without a realtime API management platform in place to maintain and manage those APIs leads to higher complexity, cost and pressure on the engineering teams to implement these crucial features from scratch.
+
+This is the gap that API Streamer fills: ready-built API management tools and a distribution network for creators of realtime APIs.
+
+h2(#publishing-data-with-api-streamer). Managing your data streams with Ably's API Streamer
+
+If you are familiar with Ably's Data Stream Network (DSN), you know that it lets you create any number of applications to stream data in realtime to any of your clients. Using the API Streamer, you can share the same data streams with developer teams at other organizations, with clear and easy to use Realtime API management features such as consolidated stats and analytics, list view of all the consumers of your streams, rate limits and such.
+
+If you already have a DSN app, you can skip to learning about the "various elements of the API Streamer":#api-streamer-elements and how to set those up.
+
+<%= partial partial_version('/api-streamer/_dsn-app-setup') %>
+
+h2(#api-streamer-elements). Understanding the elements of the API Streamer
+
+With the API Streamer, you can either publish your realtime data streams via your realtime APIs or consume a stream of data by subscribing to it, provided you have the right access.
+
+h3(#producer). Being a producer
```

Comment:
Perhaps make Producer capital.  My thinking is that it's a role, not just a word.